src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<App />, div);
```

Comment:
so, this was the default test that came in create react app. It can fail, yes, if your component is written in such a way that it doesn't render correctly. This is also a deep render, rendering every component. I think this is only really useful as a regression test.